requirements.txt

```diff
@@ -4,3 +4,4 @@ pyshp
 scipy
 cmocean
 seaborn
+sphinxcontrib-bibtex
```

Comment:
Could this be moved to docs/requirements.txt? 
To keep this requirements as clean as possible. A user who does not build the documentation, wanting only to run the model, does not need this module...